Packs/WALLIXBastion/ReleaseNotes/3_0_0.json

```diff
@@ -0,0 +1,4 @@
+{
+    "breakingChanges": true,
+    "breakingChangesNotes": null
```

Comment:
please add the breaking changes explanation inside the breakingChangesNotes field (so it will pop up for the customers as they update the version)

Reply:
Is there any tool to generate it? I saw in the docs that it expects an html string but writing html in json is not the most convenient thing to do